repos/system_upgrade/el7toel8/actors/sctpchecks/actor.py

```diff
@@ -0,0 +1,23 @@
+from leapp.actors import Actor
+from leapp.models import SCTPConfig, RpmTransactionTasks
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+class SCTPChecks(Actor):
+    """
+    Parses collected SCTP information and take necessary actions.
+
+    The only action performed by this actor is to request the installation of the
+    kernel-modules-extra rpm package, based on if SCTP is being used or not which
+    is collected on SCTPConfig message. If yes, it then produces a RpmTransactionTasks
+    requesting to install the package.
+    """
+    name = 'sctp_checks'
+    consumes = (SCTPConfig,)
+    produces = (RpmTransactionTasks, )
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        for sctpconfig in api.consume(SCTPConfig):
+            if sctpconfig.wanted:
+                self.produce(RpmTransactionTasks(to_install='kernel-modules-extra'))
```

Comment:
```suggestion
                self.produce(RpmTransactionTasks(to_install=['kernel-modules-extra']))
```

Reply:
This should have caused a Validation Error

Reply:
Yep, it causes, that's why I suggest to use list.